### ISSUE TITLE
Potential sub-lineage of BA.2.12 with ORF1ab:R1421K mutation

### ISSUE THREAD

**Issue by bikudh** [cov-lineages/pango-designation#572]
Out of approximately 66,204 BA.2.12 seen globally 4208 have the ORF1ab:R1421K mutation and they cluster together under this mutation node.  

![image](https://user-images.githubusercontent.com/88990329/164537887-6963cffe-2fb1-43d1-b11b-393309e0b02f.png)
![image](https://user-images.githubusercontent.com/88990329/164538128-438651e5-cc97-46e9-92b2-000915080fa0.png)

https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/singleSubtreeAuspice_genome_3c38e_b0fb0.json?branchLabel=nuc%20mutations&c=gt-ORF1ab_1421&label=nuc%20mutations:G4527A&tl=pango_lineage 

COV-Spectrun
https://cov-spectrum.org/explore/World/AllSamples/Past6M/variants?aaMutations=ORF1a%3AR1421K&pangoLineage=BA.2.12&

These are mostly from England, Wales and Scotland but some from India. 

@FedeGueli @AngieHinrichs @chrisruis @corneliusroemer Feel free to close the ticket if this is already assigned or an artifact. I searched but did not find any issue opened with similar observation.
Thanks!

**Comment by AngieHinrichs** [comment]
Thanks @bikudh.  From the UShER tree, it looks like those aren't really BA.2.12 -- they are adjacent to BA.2.12, in the BA.2 + C25416T branch .  (BA.2.12 is BA.2 + C25416T + C23673T/S:S704L; it seems that pangoLEARN has been calling BA.2.12 for C25416T with or without C23673T/S:S704L.)  So I believe this would be a sublineage of BA.2 (BA.2 + C25416T + G4527A/ORF1ab:R1421K).

**Comment by FedeGueli** [comment]
Good catch @bikudh it appeared first in India at the time of the first proposal of Omicron lineage! Since then it accumulated quite a lot of diversity and managed to spread in multiple countries especially in UK, 
notwithstanding its slight growth disadvantage vs BA.2 baseline in UK ( high one) i think it is worth designating this sublineage also to clarify and describe better  part of the C25416T branch not belonging to BA.2.12

**Comment by InfrPopGen** [comment]
Thanks for submitting. We've added lineage BA.2.37 with 2835 newly designated sequences, and 1126 updated designations from BA.2 (and 10 from BA.2.12). Defining mutation(s) G4527A (ORF1ab:R1421K).
